src/stores/store/eth1/local/local.go

```diff
@@ -216,10 +217,18 @@ func (s *Store) SignTypedData(ctx context.Context, addr string, typedData *core.
 	return s.Sign(ctx, addr, []byte(encodedData))
 }
 
+func (s *Store) SignEIP191Data(ctx context.Context, addr string, data []byte) (sig, msg, msgHash []byte, err error) {
```

Comment:
As we discussed it would be better to have 1:1 naming correspondency with the API, therefore it should be called SignData